### ISSUE TITLE
Research existing methods for performing image segmentation of neuroimaging data

### ISSUE THREAD

**Issue by dufaultc** [McMasterAI/Radiology-and-AI#7]
-Look at papers that have used Brats dataset for image segmentation in the past
-Write summary of different methods

**Comment by dufaultc** [comment]
Highly recommend reading [this paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7280397/) .  Provides recent and clearly explained overview of current research in using CNN's for tumour segmentation [This](https://www.sciencedirect.com/science/article/pii/S187705091632587X) paper is also worth reading to get more general overview of tumour segmentation methods

# Summary

- Three ways to perform tumour segmentation: Manual, Semi-Automatic, Fully-Automatic

- Manual is relatively accurate but suffers from high variability in results between radiologists and is time-consuming. However, manual segmentation is used to evaluate other methods

- Semi-Automatic methods are less time-consuming than manual but still suffer from high variability between radiologists

- Fully-Automatic methods can be divided into those defining custom features and CNNS

- CNN based methods have proven to be the most accurate

- One tumours are  segmented they can be further investigated to determine response to therapy, type of cancer, etc

- CNNs use convolutional layers to apply filters to input image which serve to highlight certain features of the image

- There are a few common methods used to stop the overfitting of CNNs to the input data

  - Data Augmentation: Changing the orientation and zoom of input images to stop CNN from relying on artifacts of input data

  - Dropout: Dropping nodes of the CNN to introduce network imprecision

  - Batch Normalization: Stabilizes CNN training by normalizing the affect individual nodes of model have; helpful as some may become highly biased

  -  Pooling: Downsampling the image to force CNN to learn more imprecise features of the input dataset

- Output
